en/docs/develop/hosting-authentication-endpoint-on-a-different-server.md

```diff
@@ -1,80 +1,39 @@
 # Hosting Authentication Endpoint on a Different Server
 
-The authenticationendpoint contains the authentication URLs used in
-authentication flow. You can either host
-the authenticationendpoint webapp on the WSO2 Identity Server, or choose
-to host it on a separate server. You may want to host it separately for
-the purpose of having custom theming and branding. This section
-describes how you can host the authentication endpoint on a different
-server outside the WSO2 Identity Server  (e.g., in a different Tomcat
-Server).
+The authenticationendpoint contains the authentication URLs used in authentication flow. You can either host the authenticationendpoint webapp on the WSO2 Identity Server, or choose to host it on a separate server. You may want to host it separately for the purpose of having custom theming and branding. 
 
--   [Moving the authenticationendpoint from WSO2IS and hosting it on a
-    separate web
-    server](#HostingAuthenticationEndpointonaDifferentServer-MovingtheauthenticationendpointfromWSO2ISandhostingitonaseparatewebserver)
--   [Moving the accountrecoveryendpoint from WSO2IS and hosting it on a
-    separate web
-    server](#HostingAuthenticationEndpointonaDifferentServer-MovingtheaccountrecoveryendpointfromWSO2ISandhostingitonaseparatewebserver)
--   [Running the
-    sample](#HostingAuthenticationEndpointonaDifferentServer-Runningthesample)
+This section describes how you can host the authentication endpoint on a different server outside the WSO2 Identity Server  (e.g., in a different Tomcat
+Server).
 
 ## Moving the authenticationendpoint from WSO2IS and hosting it on a separate web server
 
 !!! tip "Before you begin"
-    First, get a copy of
-    `         <IS_HOME>/repository/deployment/server/webapps        `
-    `         /authenticationendpoin.war        ` to
-    `         <WebApp_HOME>/        ` and unzip it. Make sure to get the
-    `         authenticationendpoin.war        ` after the WUM update and
-    NOT the extracted `         authentication        `
-    `         endpoint        ` in
-    `         <IS_HOME>/repository/deployment/server/webapps/        `
+    First, get a copy of the `<IS_HOME>/repository/deployment/server/webapps/authenticationendpoint.war` file to `<WEBAPP_HOME>/` and unzip it. Make sure to get the `authenticationendpoint.war` after the WUM update and NOT the extracted `authenticationendpoint.war` in the `<IS_HOME>/repository/deployment/server/webapps/` folder.
```

Comment:
In latest versions `authenticationendpoint.war` is no longer there. Instead, extracted `authenticationendpoint` folder is shipped. Change this line accordingly